src/sentry/utils/cloudfunctions.py

```diff
@@ -18,10 +18,45 @@
 WRAPPER_JS = """
 const userFunc = require('./user.js');
 Object.assign(process.env, require('./env.json'));
+const Sentry = require("@sentry/node");
+const Tracing = require("@sentry/tracing");
+exports.start = async (message, context) => {
+  if (process.env.SENTRY_DSN) {
+    Sentry.init({
```

Comment:
Important: Just a heads up, make sure `Sentry.init()` is not called within the function/handler itself but in the global scope of a GCP function file.

The reason is that if a function is hot when it gets hit, the SDK is initialized multiple times in the runtime - once for the time the function was cold + any number of additional times for when the function was hot. Multiple init calls in the same global context cause all kinds of breakage and slowdown (stuff being wrapped over and over, breadcrumbs being recorded twice, ...).